plugins/formal-verification/testData/diagnostics/bad_contracts/calls_in_place_leak.fir.diag.txt

```diff
@@ -52,7 +51,6 @@ method global$fun_function_object_call$fun_take$fun_take$fun_take$$return$T_Unit
   label label$ret$0
 }
 
-/calls_in_place_leak.kt:(544,564): warning: Viper verification error: Assert might fail. Assertion special$duplicable(local$g) might not hold. (<no position>)
-
+/calls_in_place_leak.kt:(544,564): warning: Viper verification error: [assert.failed:assertion.false] Assert might fail. Assertion special$duplicable(local$g) might not hold. (PositionWrapper(ktPosition=org.jetbrains.kotlin.KtLightSourceElement@9427c1a7))
```

Comment:
It would be nice to have a more friendly output here; in particular, it's hard to check that the output is correct like this.